.github/dependabot.yml

```diff
@@ -11,7 +12,8 @@ updates:
       - "dependencies"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories:
+      - "**/*"
     schedule:
```

Comment:
Shouldn't only the "directories" directive be applied here? And not in the "pip" section, consequently

Reply:
Yeah, that's a good point. Let me restore the lines for the pip ecosystem.

Reply:
Great thanks!